lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -32,6 +32,15 @@ def mock_valid_query(**kwargs):
     return create_user()
 
 
+@pytest.fixture(autouse=True)
+def server_config_env(on_disk_server_config, monkeypatch):
+    """Provide a pbench server configuration environment variable for all user
+    management CLI tests.
+    """
```

Comment:
Are you sure that you want this fixture to be `autouse`'d?  Wouldn't it be better to expect/require those tests which actually need/want environment-variable-based access to the config file to explicitly reference the fixture?

Reply:
All CLI command invocations look for the environment variable.